js/video-controller.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Class representing the main menu navigation.
+ */
+export default class VideoController {
+  constructor(classEl) {
+    let videoWrapper = document.querySelector(
+      "." + classEl + "__controller-wrapper"
+    );
+
+    if (videoWrapper) {
+      let video = videoWrapper.getElementsByTagName("video")[0];
```

Comment:
Instead of a `let video`, let's do a `this.video`, as that will make the video an actual property of the object. You'll need to adjust your references on lines 15 and 16 to `this.video` accordingly.